recipes/pyro4/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "4.43" %}
+
+package:
+  name: pyro4
+  version: {{ version }}
+
+source:
+  fn: Pyro4-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/P/Pyro4/Pyro4-{{ version }}.tar.gz
+  md5: 5c094edfdd465ed52fdf9b2a4f806f74
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - pyro4-ns = Pyro4.naming:main
+    - pyro4-nsc = Pyro4.nsc:main
+    - pyro4-test-echoserver = Pyro4.test.echoserver:main
+    - pyro4-check-config = Pyro4.configuration:configuration_dump
+    - pyro4-flameserver = Pyro4.utils.flameserver:main
+    - pyro4-httpgateway = Pyro4.utils.httpgateway:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - serpent >=1.11
+
+  run:
+    - python
+    - serpent >=1.11
+
+test:
+  imports:
+    - Pyro4
+    - Pyro4.socketserver
+    - Pyro4.test
+    - Pyro4.utils
+
+  commands:
+    - pyro4-ns --help
+    - pyro4-nsc --help
+    - pyro4-test-echoserver --help
+    - pyro4-check-config --help
+    - pyro4-flameserver --help
+    - pyro4-httpgateway --help
+
+about:
+  home: http://pythonhosted.org/Pyro4/
+  license: MIT
+  summary: 'Distributed object middleware for Python (RPC)'
+
+extra:
+  recipe-maintainers:
+    - whart222
+    - irmen
+    - Juanlu001
```

Comment:
Sorry. It was buried in an outdated comment from 23 days ago. Looks like you were happy then. Hoping that is still true. 😄